### ISSUE TITLE
Standard overlay confusion

### ISSUE THREAD

**Issue by blelump** [the-human-colossus-foundation/oca-spec#45]
### Problem overview

When we think of a standard, there are many viable options. There are ISO standards, IETF standards, W3C standards, etc. Furthermore, IETF standards, for example, did not pass a formal review process in many cases, even though they are broadly used in the industry. 

A standard brings a narrow specialization for a domain-oriented problem and offers a solution. In the OCA ecosystem, the standard is additional metadata that currently does not add any value to the ecosystem except the ability to define it as pure metadata. The primary implementation does not benefit from it. It is, therefore, a burden we need to carry for reasons that need to be clarified.

Specifying a standard and, for example, imposing the behavior upon the standard is impossible due to the amount of existing standards. The vague concept of this overlay and its lack of applicability suggest the DSWG consider either redefining its meaning and purpose or removing it entirely.

**Comment by carlyh-micb** [comment]
Recommend that standard overlay be removed and functionality replaced with the more accessible and descriptive entry_framing overlay.

Similar to other framing overlays (see RFC 004 https://github.com/the-human-colossus-foundation/oca-spec/blob/master/RFCs/0004-framingOverlay.md):

{
  "capture_base": "Etszl9LgLUjllI950rd2lO6rF5-BP_jGzXGBPkFZCZFA",
  "digest": "XXXX",
  "type": "spec/overlays/entry_framing/1.0",
  "entry_framing": {
    "chemical": {
      "frame_id": "https://doi.org/10.1515/iupac",
      "frame_label": "IUPAC nomenclature",
      "frame_location": "https://iupac.org/what-we-do/nomenclature/",
      "frame_version": ""
    }
  }
}

Note IUPAC nomenclature contains the rules for writing chemical formulas and there are an infinite number of combinations that could be created and thus it is impossible to write this as a list of entry codes.

**Comment by carlyh-micb** [comment]
I don't think this belongs as entry_framing but should continue to be standard.

```
{
  "capture_base": "Etszl9LgLUjllI950rd2lO6rF5-BP_jGzXGBPkFZCZFA",
  "digest": "XXXX",
  "type": "spec/overlays/entry_standard/1.0",
  "entry_standard": {
    "chemical": {
      "standard_id": "https://doi.org/10.1515/iupac",
      "standard_label": "IUPAC nomenclature",
      "standard_location": "https://iupac.org/what-we-do/nomenclature/",
      "standard_version": ""
    }
  }
}
```

**Comment by carlyh-micb** [comment]
DSWG has started an RFC to change this standard and update it. We absolutely want it to remain in the specification so that we don't overload other overlays with information that can be more easily sorted into overlays such as the standard overlay.

https://github.com/the-human-colossus-foundation/oca-spec/blob/RFC-006-standard/RFCs/0006-Standard_overlay_changes.md

**Comment by carlyh-micb** [comment]
I've completed the RFC with Paul and I think I have put in a request to merge into master but I might be wrong - still learning GitHub. If I did it wrong please help me do it right the next time. Thank you!

**Comment by carlyh-micb** [comment]
Today we discussed framing and standard overlays in DSWG.

Initially I agreed that framing overlays for entry could replace standard overlays.
However, I was thinking more carefully about examples and realized a few points.

When a data entry record is written to a standard it doesn't mean it is therefore framed to a concept.
For example, I can describe this chemical compound with its IUPAC name.
2-Acetoxybenzoic acid (which is the standard IUPAC name for aspirin)

This is not a concept. It is just a chemical name. It is written according to a standard. There are no associated definitions to this chemical name, it is not in a hierarchy of knowledge. It is just a name written to a standard.

Post-hoc after it is recorded as a data entry, I could then frame 2-Acetoxybenzoic acid to the concept of:
https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=CHEBI:35475
Which is a non-steroidal anti-inflammatory drug (NSAID) where (2-Acetoxybenzoic acid broadMatch CHEBI:35475) (aspirin is a subset of NSAIDs). I might perform this framing to understand what the influence of drug type has on my results through my data analysis.

Some standards may give context because the selected standard is very contextual, but not all standards are contextual. Standards is a broader class than framing.

Therefore - I propose that it should be entry standards instead of entry framing, where the records entered according to a standard may need post-analysis to frame to some other knowledge source that has contextual information.

**Comment by pknowl** [comment]
@carlyh-micb Yes, that makes sense. Standards and framing serve distinct roles within the OCA ecosystem, addressing unique aspects of data management. Standards focus on structural rules or expected formats for entered values, such as adhering to IUPAC naming conventions. These fall under morphological semantics, emphasizing how data is represented and structured. In contrast, framing connects structural attributes to conceptual knowledge, such as mapping an IUPAC name to its corresponding ontology term in ChEBI, engaging epistemological semantics to emphasize meaning and relationships. By keeping these overlays separate, standards ensure uniformity in data structure, while framing fosters semantic interoperability.

This separation enhances clarity of purpose, allowing users to differentiate between ensuring compliance with specific formatting standards and adding semantic context for richer interoperability. Combining these functions into one overlay risks creating confusion and blurring their distinct purposes, particularly when standards do not inherently imply a connection to broader concepts. For example, ISO 8601, a standard for recording dates, focuses purely on format without requiring conceptual links.

The modular nature of the OCA ecosystem supports this distinction. A dedicated standards overlay complements the framing overlay, aligning with principles of modularity and separation of concerns. This ensures that each overlay remains extensible and well-suited to its specific role within the system.

**Comment by carlyh-micb** [comment]
And to add some further examples:

I may enter data free form in data records, and I can create entry codes without following any standard.

Or I can enter data or create entry codes that follow a specific standard.

Independent of the standard, I may frame my entries into one or more ontologies or other type of knowledge graph.
I can only frame entries when I know what they are. Therefore, only post-hoc will I be able to frame any freely entered entry. If I followed a standard it could help me with post-hoc framing, but it might not.

For example: I could create an entry code overlay that follows ISO-standard two letter country codes. I could also frame those countries into an ontology that connects countries by geography so I could reason in my analysis based on location. I could also frame those countries into an ontology that gives the dominant languages in each country so I could also reason in my analysis based on language.

I could create an entry code overlay that doesn't follow ISO-standard country codes, and still connect it to an ontology. This sounds silly (because the country codes exist), but, for example, in disease surveillance I could be tracking both well known diseases (and therefore likely present in an ontology) as well as emerging diseases. Emerging diseases may have a community ad-hoc name established but are not yet officially recognized in the ontology and the formal, standardized name has not yet been established (e.g. formal names in biology have rules for acceptance and a process that must be followed). Therefore I can create entry code lists that don't contain only standardized names. I could map all these terms to an ontology (perhaps the new diseases may be broadMatch mapped to the viral family for example). Ontologies and standards are always lagging when adding terms.

Therefore I propose that we need both: standards overlay and framing overlay, because they serve different and distinct purposes.

**Comment by blelump** [comment]
How does it relate to https://github.com/the-human-colossus-foundation/oca-spec/pull/64 ?

**Comment by pknowl** [comment]
@blelump The above dialogue reinforces that we must keep two separate overlay types for (i.) **Framing** and (ii.) **Standards**.  

RFC 0004 describes the Framing Overlay 
https://github.com/the-human-colossus-foundation/oca-spec/blob/master/RFCs/0004-framingOverlay.md  

RFC 0006 describes the Standard Overlay 
https://github.com/the-human-colossus-foundation/oca-spec/blob/master/RFCs/0006-Standard_overlay_changes.md  

@carlyh-micb - Anything else to add?

**Comment by carlyh-micb** [comment]
#64 does not require any changes.

There was a discussion with Robert in last DSWG meeting that framing could have an additional overlay to replace standard, but I give examples here to support the idea that standard overlay is the correct way to proceed and therefore #64 does not need changes. 

Paul's comment is accurate.
